docs/metrics/metrics.md

```diff
@@ -0,0 +1,43 @@
+# Metrics
+
+The Pactus blockchain offers [Prometheus](https://prometheus.io/) metrics for its network module, enabling users to monitor and analyze various network-related statistics. To activate this feature, inside the `config.toml`,  set the `enable_metrics` parameter to true.  Also, ensure that the HTTP module is enabled. You can enable HTTP module under the `[http]` section of the `config.toml` file. Once enabled, the metrics can be accessed at [http://localhost:8080/metrics/prometheus](http://localhost:8080/metrics/prometheus).
+
+After these changes, restart the Pactus node; you should now be able to view the metrics.
+
+## Prometheus Configuration
+
+Prometheus is an open-source monitoring and alerting tool that facilitates the collection and processing of metrics. A common method of running Prometheus is via Docker containers. To use Prometheus with Docker, follow these steps:
+
+1- Ensure [Docker](https://www.docker.com/) is installed on your system.
+
+2- Pull the Prometheus Docker image:
+
+```text
+docker pull prom/prometheus
+```
+
+3- Create a configuration file named `prometheus.yml` to define the Prometheus configuration. You can refer to the Prometheus [documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) for more guidance. As an example, here's a simple configuration:
+
+```yaml
+global:
+  scrape_interval: 1m
+
+scrape_configs:
+  - job_name: "prometheus"
+    scrape_interval: 1m
+    static_configs:
+      - targets: [ "127.0.0.1:9090" ]
```

Comment:
We should change the port. because right now we use port 9090 for gRPC. Look [here](https://github.com/pactus-project/pactus/blob/236b5cac9e124a3baea8162478ae01c958e2f607/config/config.go#L116)